hibike/hibike_tester.py

```diff
@@ -0,0 +1,35 @@
+import hibike_process
+from multiprocessing import Process, Pipe, Queue
+
+class Hibike:
+
```

Comment:
Which blank lines were you referring to? 3, 5, or 7? I've deleted 7 but kept 3 and 5 for now